build.gradle

```diff
@@ -116,9 +117,16 @@ clean.doFirst {
 tasks.build.dependsOn "compressTiny", "tinyJar", "v2UnmergedMappingsJar", "v2MergedMappingsJar"
 
 task insertAutoGeneratedMappings(dependsOn: [buildMappingsTiny, downloadPerVersionMappings], type: AddProposedMappingsTask) {
+	final enigmaProfile = file("enigma_profile.json")
+
+	inputs.file(
+		new JsonSlurper().parse(enigmaProfile)
+			.services.jar_indexer.args.simple_type_field_names_path
+	)
+
```

Comment:
This... Works for now. I would make an issue once this is merged to do this properly.

Reply:
afaict a proper fix would require enigma plugins to have a way of listing files they depend on, then enigma would list all files all plugins depend on

Is that what you mean?
Or something in qmap buildSrc?

Reply:
something in the buildsrc